### ISSUE TITLE
Tests "V8Js::executeString" (2) is failing with V8 >= 5.3.337

### ISSUE THREAD

**Issue by AlexMasterov** [phpv8/v8js#240]
```
PHP  7.0.8
V8   5.3.337 / 5.4.3
V8Js 1.3.0
```
```
FAILED TEST SUMMARY
---------------------------------------------------------------------
Test V8::executeString() : Forcing to arrays (argument passing) [tests/execute_flags_args.phpt]
Test V8::executeString() : Prototype with PHP callbacks [tests/object_prototype.phpt]
```

**Comment by stesie** [comment]
Fixed in V8Js releases 0.6.4 (PHP 5) and 1.3.1 (PHP 7), thanks for reporting, I had not noticed it before

**Comment by AlexMasterov** [comment]
@stesie 

Thanks for fixing this so quickly!
